mobile-app/app/screens/AppNavigator/screens/Balances/screens/SendScreen.tsx

```diff
@@ -128,8 +127,8 @@ export function SendScreen ({
     setHasBalance(totalBalance.isGreaterThan(0))
   }, [JSON.stringify(tokens)])
 
-  useEffect(() => {
-    setBottomSheetScreen([
+  const bottomSheetScreen = useMemo(() => {
```

Comment:
Let's remove `useMemo` here, might be a potential cause of the disappearing bottom sheet

Reply:
adding `useMemo()` is actually to fix disappearing bottom sheet